### DISCUSSION TITLE
Grid does not work astro

### DISCUSSION THREAD

**Discussion by DD10654** [tailwindlabs/tailwindcss#9687] [Help]
Hello friends, I need some help using Tailwind with Astro

I am creating a box with buttons inside them, I want 3 in one line and the others to go to the 2nd, 3rd, 4th, etc. Here is the code I tried with the result

Code :

![Screenshot 2022-10-29 at 9 30 13 AM](https://user-images.githubusercontent.com/75024070/198813250-dedb27cc-50aa-4821-812c-89e45c4c45b1.png)

Result :

![Screenshot 2022-10-29 at 9 40 03 AM](https://user-images.githubusercontent.com/75024070/198813264-0ff5a170-2f9d-47a9-832c-1f7142f4722b.png)

**Comment by DD10654** [comment]
I am using grid-cols-3 yet the buttons all stay in the same line

**Reply by hpy688** [reply]
Try adding "flex-col" to the classname of the div tag

**Reply by hpy688** [reply]
Maybe the flex direction is wrong

**Reply by wongjn** [reply]
It seems like the `grid-cols-3` class is not being generated in the CSS. Consider checking for any misconfiguration.

**Reply by DD10654** [reply]
I checked the config, and it all seems fine though as the style for the buttons remains the same, I also tried flex-col, yet it did not work. I tried the same code on tailwind play and it worked. I removed the section tag that the div too, but it did not work. Thank you for trying tho !

**Reply by wongjn** [reply]
If you can provide the repository to the project or a repository that produces the same issue, we could take a closer look for you if you would like.

**Reply by DD10654** [reply]
Hello, @wongjn I tried manually adding the CSS instead of using tailwind and it works perfectly. For me it is ok now, but if you want to experiment with it, here's the repo link: https://github.com/DD10654/accessible-astro-dashboard

**Reply by wongjn** [reply]
Thank you for providing the repository. I have taken a look and it is [this custom rule](https://github.com/DD10654/accessible-astro-dashboard/blob/576ea8ea570e36f37fab57e2f6c2be71faa800da/src/assets/scss/base/_grid.scss#L28) that is overriding Tailwind's `grid` class. Consider removing this rule, renaming the class name this selector uses, or add a `prefix` configuration to Tailwind's classes to avoid conflicts.
